improve_C1 branch Game screen improved version 확인 요망

1000peach opened the pull request:
v0.2.0 Game screen improved version 변경 사항
- console size 조절
- printBoard 함수의 내용 구현: 잘못된 출력 제거, 함수화를 통해 main.c 에서의 중복 제거
- textcolor 함수를 통해 숫자 별로 색 부여
- 전체적인 branch의 개선 목적: 게임화면을 시각적으로 편리하게 개선

총 3 commits의 게임 화면 개선 버전 improve_C1 입니다.
더 자세한 변경 사항은 commit message에 적어두었습니다.
확인하시고 수정사항 또는 오류가 발생하면 코멘트로 남겨주세요!
@LeeEunhyung @wlals7217

LeeEunhyung commented:
improve_C1 개선사항 확인했습니다!
그런데 InitGame.c에서 여는 괄호 부분을 정한 규칙대로 해주시면 좋을 것같아요!
또 menu 비교할 때 상수 처리하는 것도 좋을 것같습니다!
그리고 이거는 제가 개선해야할 사항인데 Windows.h가 Windows OS에서만 실행 가능해서 해당 사항은 개선 환경에 맞게 수정하겠습니다!
수고하셨습니다!~!

1000peach commented:
> improve_C1 개선사항 확인했습니다!
> 그런데 InitGame.c에서 여는 괄호 부분을 정한 규칙대로 해주시면 좋을 것같아요!
> 또 menu 비교할 때 상수 처리하는 것도 좋을 것같습니다!
> 그리고 이거는 제가 개선해야할 사항인데 Windows.h가 Windows OS에서만 실행 가능해서 해당 사항은 개선 환경에 맞게 수정하겠습니다!
> 수고하셨습니다!~!

네, menu 상수 처리하고, 글자색에 필요한 상수들도 고려할게요!
중괄호 지키지 못한 부분까지 수정하여 내일 중으로 다시 올리겠습니다!
리눅스 개선 방법도 같이 찾아보고 있을게요! 코멘트 감사합니다~

1000peach committed:
Change console size

- console size를 게임 화면에 맞게 조절해 원활한 게임 환경 개선

1000peach committed:
Make function (printBoard)

- printBoard: gameboard, 상태 출력 부분 함수화-> 잘못된 출력 제거, 간단히 구현

1000peach committed:
Improve game screen

- printBoard 개선: 키 안내 출력을 한 눈에 들어오게 수정,
  gameboard에서 숫자 별로 색 구분
- textcolor 함수 추가: 숫자에 색 부여하는 기능
- 전체적으로 시각적으로 편리한 game screen 개선
